core/src/main/scala/cats/data/NonEmptyList.scala

```diff
@@ -0,0 +1,68 @@
+package cats
+package data
+
+import cats.std.list._
+//it needs a Functor
+//it needs semigroup - combine, filter
+
+/**
+  *  A data type which represents a single element (head) and some other
+  * structure (tail).
+  */
+final case class NonEmptyList[A](head: A, tail: List[A]) {
+
+  /**
+    * Return the head and tail into a single list
+    */
+  def unwrap: List[A] = head :: tail
```

Comment:
I think in `OneAnd` it's called `unwrap` since `F` is abstract so there's not a straightforward name to call it. Since we are fixed to `List` here, it's easier to come up with a good name :)